localization/languages/hu.json

```diff
@@ -88,8 +88,8 @@
     "timeRangeJustNow": "Idő intervalum, éppen most",
     "timeRangeMinutes": "Idő intervalum, perc",
     "timeRangeHour": "Idő intervalum, óra",
-    "timeRangeToday": null, //missing translation
-    "timeRangeYesterday": null, //missing translation
+    "timeRangeToday": "Idő intervalum, ma", //missing translation
+    "timeRangeYesterday": "Idő intervalum, tegnap", //missing translation
```

Comment:
I think these should just be "ma" / "tegnap"?

(If so, the existing translations are wrong also - would you mind fixing those?)